src/Control/Distributed/Spark/RDD.hs

```diff
@@ -119,6 +158,27 @@ aggregate seqOp combOp zero rdd = do
   res :: JObject <- call rdd "aggregate" [coerce jzero, coerce jseqOp, coerce jcombOp]
   reify (unsafeCast res)
 
+-- | A version of aggregate implemented in terms of 'mapPartitions'.
+aggregate'
```

Comment:
Revise documentation and naming.

As things stand, the documentation gives me no idea why I would want to use `aggregate'` over `aggregate`. They both have the same type signature and pretty much the same spec, just a different implementaiton.

I object to adding ad hoc variations of Spark API functions in our set of bindings, unless there is a clear and problematic gap in Spark's API that we ought to fill in ourselves. But in that case we should explain in the documentation what the gap is. `aggregate'` probably deserves a more descriptive name - tacking a prime to `aggregate` doesn't say much.